### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by xpple**

### CODE REVIEW

File: src/main/java/net/earthcomputer/clientcommands/command/WhisperEncryptedCommand.java [hunk 1]
```diff
@@ -34,73 +38,66 @@
...
+            cipher.update(compressedBytes);
+            compressedBytes = cipher.doFinal();
+            String commandMessage = "w " + recipient.getProfile().getName() + " CCENC:" + BaseUTF8.toUnicode(compressedBytes);
             if (commandMessage.length() >= 256) {
-                source.sendError(Text.translatable("commands.cwhisperencrypted.messageTooLong"));
-                return 0;
+                throw MESSAGE_TOO_LONG_EXCEPTION.create();
             }
+            Text preview = Util.map(source.getClient().inGameHud.getChatHud().getChatScreen().getChatPreviewer().tryConsumeResponse("/" + commandMessage), ChatPreviewer.Response::previewText);
+            source.getPlayer().sendCommand(commandMessage, preview);
```
Comment (wagyourtail) [inline src/main/java/net/earthcomputer/clientcommands/command/WhisperEncryptedCommand.java]: You dont need to create the preview iirc. Its `@nullable`

**Review by wagyourtail — CHANGES_REQUESTED** [review]

File: src/main/java/net/earthcomputer/clientcommands/command/WhisperEncryptedCommand.java (hunk 1 above)
Comment (xpple) [reply src/main/java/net/earthcomputer/clientcommands/command/WhisperEncryptedCommand.java]: Right. It used to be omitted though, which didn't work anymore.

Comment (wagyourtail) [reply src/main/java/net/earthcomputer/clientcommands/command/WhisperEncryptedCommand.java]: So passing null doesnt work anymore?

Comment (xpple) [reply src/main/java/net/earthcomputer/clientcommands/command/WhisperEncryptedCommand.java]: No not passing `null`, omitting the argument. I am currently testing passing `null`.

Comment (xpple) [reply src/main/java/net/earthcomputer/clientcommands/command/WhisperEncryptedCommand.java]: Passing `null` seems to work, I will change it.

**Pull request merged**